### DISCUSSION TITLE
804

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#980] [Comments]
# 804

스즈키 히로키의 "마키아벨리의 군주론을 읽다"를 읽고 생각나는 구절들을 정리했다.1장구두쇠가 되어도 좋다"군주는 인색하다는 평판에 개의치 않아도 된다.백성을 지키기 위해서도, 자신을 지키기 위해서도 (중략) 그래야 한다.구두쇠가 되는 것은 지배자가 자신의 자리를 보존하기 위한 하나의 악덕이다."...낭비하는 리더는 큰일을 치를 때마다 국민에게 피해를 주고 원망을 받는 반면에, 절약하는 리더는 국민의 지지를 받으면서 큰일을 해냅니다.유동성이 풍부한 스타트업 투자 호황기의 대표님들이 생각난다.그 시절에 낭비를 했던 리더분들은 대단히 힘든 시기를 보내었고,당시에 투자금을 필요한 곳에만 적절하게 사용하신 분들은 지금 혹한기 시절에 오히려 더 성과를 내고 있다.인건비의 과도한 투자로 요즘의 시기가 어렵다는 뉴스가..

https://jojoldu.tistory.com/804

**Comment by msbaek** [comment]
"군주는 야망을 이루기 위해 때로는 천성을 버려야 합니다" 이 문장이 가장 인상적이네요.

그리고 개발조직을 줄여서 비용 구조를 개선해서 매출, 영업이익을 모두 높일 수 있다는 것은 알겠는데요.
향후 성장 가능성을 고려하면 무조적 그래야 하는 것은 아닐 것 같다는 생각이 듭니다.
코로나 시기나 경기가 좋았을 때 IT쪽에 과하게 투자가 되었다고는 생각합니다만...

어렵습니다 ㅠㅠ

**Reply by jojoldu** [reply]
물론입니다 명석님! ㅎㅎ 
저도 소프트웨어가 주력인 환경에서는 당연히 소프트웨어 개발에 대한 투자를 계속 해야한다고 생각하는데요.

다만, 예전에는 VC분들도 업종 불문 투자금으로 소프트웨어 엔지니어를 채용하라고 했다면,
지금은 업종에 따라 엔지니어에 대한 투자 관련된 이야기가 완전히 상반되더라구요 ㅎㅎ

영록님 트위터 보면서 들었던 생각은, 
예전에는 회사의 스테이지, 회사의 비즈니스 모델에 상관없이 소프트웨어 개발자에 대한 투자가 왕성했다면
이제는 지금 우리 회사의 비즈니스 모델, 스테이지에 따라 중요한 것이 무엇인지 고민해야하고, 
회사의 핵심이 무엇이냐에 따라 네이버 스마트스토어가 더 적합할지, 자체 개발팀을 꾸리는게 더 적합할지, 우리 스테이지에서 어떤것이 가장 좋을지 대표님들도 고민하는 시대가 온 것 같다는 생각이 들었어요 ㅎㅎ

댓글을 쓰다보니 요즘 같은 시기에서는 개발팀은 회사에 어떤 역할을 해야할까에 대해 고민이 많이 필요한 시대가 된 것 같다는 생각이 드네요 :)
